### ISSUE TITLE
upgrade piwik

### ISSUE THREAD

**Issue by rondepierre** [piwik/piwik#10383]
hi all,

since the last  security upgrade ( 2.16.2 ), we have back this old issue: ( I saw this issue on old bug track )
Maximum recursion level of 15 reached. Maybe you have set a DataTable\Row with an associated DataTable belonging already to one of its parent tables

regards

**Comment by tsteur** [comment]
Do you maybe have URLs that have many "directories" in the path? 

When you say it is "back". Have you created an issue before about this where we fixed something?

**Comment by rondepierre** [comment]
hi tsteur,

   before the upgrade, I had no error.
   I did not add/update/delete any accounts on my piwik.
   after the upgrade, these messages came.
   I will check if during the upgrade DataTable\Row makes some issues.

When you say it is "back". I saw on old Git ticket that one similar message, and it was solved. 
this is why I wrote, back .

I will check the DataTable\Row, I will comeback to you

regards
